roles/ansible_service_broker/defaults/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 
 ansible_service_broker_node_selector: "{{ openshift_hosted_infra_selector | default('region=infra') | map_from_pairs }}"
+ansible_service_broker_enable_dashboard_redirector: false
+ansible_service_broker_dashboard_redirector_route: "dr-1337-openshift-ansible-service-broker-{{ openshift_master_default_subdomain }}"
```

Comment:
Is this route correct? Think if would form `dr-1337-openshift-ansible-service-broker-example.com` unless I'm reading it wrong.

Reply:
Oops, should be `.{{ openshift_master_default_subdomain }}`